### ISSUE TITLE
`navigate` in Cloudflare Workers (esbuild) throws `glob is not a function`

### ISSUE THREAD

**Issue by dan-lee** [brillout/vite-plugin-ssr#301]
We are running a project with Cloudflare Worker built with `esbuild`.
It works fine but when we import and call [`navigate()`](https://vite-plugin-ssr.com/navigate#page-content) we get this error:

```
TypeError: (intermediate value).glob is not a function
    at node_modules/vite-plugin-ssr/dist/esm/client/page-files/pageFiles.js (/.../vite-ssr-project/dist/worker.js:9171:28)
    at __init (/.../vite-ssr-project/dist/worker.js:15:56)
    at node_modules/vite-plugin-ssr/dist/esm/client/page-files/setup.js (/.../vite-ssr-project/dist/worker.js:9183:5)
    at __init (/.../vite-ssr-project/dist/worker.js:15:56)
    at node_modules/vite-plugin-ssr/dist/esm/client/router/index.js (/.../vite-ssr-project/dist/worker.js:11033:5)
    at __init (/.../vite-ssr-project/dist/worker.js:15:56)
    at dist/server/assets/index.page.413c48c9.js (/.../vite-ssr-project/dist/worker.js:11046:19)
    at __require2 (/.../vite-ssr-project/dist/worker.js:18:50)
    at /.../vite-ssr-project/dist/worker.js:21034:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Repro: https://github.com/dan-lee/vps-navigate-esbuild-bug-repro

Run `yarn preview:miniflare` and open the page.

**Comment by brillout** [comment]
Will release in a moment.

**Comment by dan-lee** [comment]
Thank you for the quick fix 🙏

**Comment by brillout** [comment]
Released in `0.3.62`.
